Add brief Tobira "documentation" (#7116)
opencast/opencast: docs/guides/admin/docs/configuration/tobira.md

Tobira
======

Tobira is a video portal for Opencast that aims to be a pleasant interface through which users interact with Opencast content.

Videos, series and playlists can be presented in a customizable, hierarchical page structure, and users can easily search through all media. Additionally, Tobira offers tools to upload videos and create series, as well as to edit some metadata. This is currently limited to changing the title, description, access policies and series content.

Tobira's main goal is to serve as a video portal for end users.
**It is not meant to replace the Admin interface.**

> That should still be the preferred address for **admins** to manage publications.

Reviewer: This could be cleaned up a bit to read more like this

```suggestion
**It is not meant to replace the Admin interface, which is still be the preferred address for admins to manage publications.**
```

I'm still not happy with it, but it cleans up the `That`, which was somewhat unclear.

Any changes done in Tobira are immediately sent to Opencast, while changes done in Opencast are synced periodically to prevent conflicts.

## Setup and configuration

Tobira runs separately from Opencast and needs its own installation. The configuration necessary to connect these systems is mainly done in Tobira itself, but there are some things that you'll need to adjust in Opencast as well. For an in-depth explanation and the complete setup and configuration guide, see Tobira's main [documentation](https://elan-ev.github.io/tobira/setup).

## Tobira in the Admin UI

> When you have connected a Tobira instance to your Opencast, the Admin UI offers some additional features: When creating a new series, you can choose in which Tobira page it should be mounted. That path can be also be changed or initialized later on.

Reviewer: ```suggestion
When you have connected a Tobira instance to your Opencast, the Admin UI's series modal will now allow you to choose the series' mount point in Tobira.
```

If there are docs on what a `mount point` is, I would link to those.

Author: Unfortunately, those don't really exist aside from some doc comments. I added a short explanation in brackets.
